packages/teams-js/src/public/pages.ts

```diff
@@ -232,7 +232,7 @@ export namespace pages {
    */
   export function registerFullScreenHandler(handler: fullScreenChangeFunctionType): void {
     registerHandlerHelper('fullScreenChange', handler, [], () => {
-      if (handler && !isSupported()) {
+      if (handler !== undefined && !isSupported()) {
```

Comment:
>handler !== undefined [](http://example.com/codeflow?start=10&length=21)

Using an explicit `undefined` check rather than the implicit `if (handler)` check makes it clear to the compiler of the intent (without that check, the compiler would question whether the intent was calling `handler()`)